src/schema/index.js

```diff
@@ -157,8 +160,11 @@ const schema = new GraphQLSchema({
       updateCollectorProfile: UpdateCollectorProfile,
       updateMyUserProfile: UpdateMyUserProfileMutation,
       updateConversation: UpdateConversationMutation,
-      updateOrder: UpdateOrderMutation,
+      confirmOrder: ConfirmOrderMutation,
+      finalizeOrder: FinalizeOrderMutation,
+      rejectOrder: RejectOrderMutation,
       submitOrder: SubmitOrderMutation,
+      updateOrder: UpdateOrderMutation,
```

Comment:
with these more specific mutations, do we still need `updateOrderMutation`?

Reply:
Most likely not. `updateOrder` and `submitOrder` were there from another project for checkouts in emission which didn't get pushed to production. After finalizing MP API we can remove/refactor them.

PR here: https://github.com/artsy/metaphysics/pull/1009

Reply:
This isn't being used in production, so you can feel free to remove.

Reply:
(actually let me verify-- I know it's not being _consumed_ but I'm not sure if its presence here will break anything? I feel like this is auto-generated https://github.com/artsy/reaction/blob/f765cb4a46e3f68c62de243ad193f68e69582741/data/schema.graphql#L5609)

Reply:
Yes I think so. Emission also has a similar file that is the output of MP `dump-schema` copied using `sync-schema` [here](https://github.com/artsy/reaction/blob/master/package.json#L43).